lib/install/management/inspect.go

```diff
@@ -140,41 +149,31 @@ func (d *Dispatcher) InspectVCH(vch *vm.VirtualMachine, conf *config.VirtualCont
 	return nil
 }
 
-// verifyClientCert checks whether the input path has a client cert that's valid for the VCH.
-func verifyClientCert(ca []byte, path string) error {
+// VerifyClientCert verifies the loaded client cert keypair against the input CA and
+// returns the certificate on success.
+func VerifyClientCert(ca []byte, ckp *certificate.KeyPair) (*tls.Certificate, error) {
```

Comment:
Based on our offline discussion, this func is now generic enough to belong in `pkg/certificate`.